File: app/src/main/java/com/hover/stax/destruct/SelfDestructActivity.java
package com.hover.stax.destruct;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.hover.stax.databinding.SelfDestructLayoutBinding;
import com.hover.stax.utils.Utils;

public class SelfDestructActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        SelfDestructLayoutBinding binding = SelfDestructLayoutBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        binding.continueBtn.setOnClickListener(view -> Utils.openStaxPlaystorePage(this));
    }
}

File: app/src/main/java/com/hover/stax/destruct/SelfDestructActivity.kt
package com.hover.stax.destruct

import com.hover.stax.utils.Utils.openStaxPlaystorePage
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.hover.stax.databinding.SelfDestructLayoutBinding

class SelfDestructActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val binding = SelfDestructLayoutBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.continueBtn.setOnClickListener { openStaxPlaystorePage(this) }
    }
}